### DISCUSSION TITLE
MMOCR 1.x dataset support CTW 1500

### DISCUSSION THREAD

**Discussion by jorie-peng** [open-mmlab/mmocr#1605] [General]
In MMOCR textdet document, only publish CTW 1500 and IC15 log and hmean, but 1.x cannot support CTW 1500, can you support 1.x dataset CTW 1500? or publish other supported datasets log and hmean for textdet。Thx！

**Comment by gaotongxiao** [comment]
It's supported, but not in a fully automatic way. The steps here are still working:
https://mmocr.readthedocs.io/en/dev-1.x/user_guides/data_prepare/det.html#ctw1500

**Reply by jorie-peng** [reply]
this method is for MMOCR 0.x, I will try to translate it to 1.x，Thanks for your help!

**Comment by Harold-lkk** [comment]
Temporarily， CTW1500 is not supported by data prepare. You can use https://github.com/open-mmlab/mmocr/blob/dev-1.x/tools/dataset_converters/textdet/ctw1500_converter.py to prepare CTW1500 right now. We will support CTW1500 by DataPreprepare ASAP. 

If possible， we can implement it together!

**Reply by jorie-peng** [reply]
ok, Thanks for your help! I will try it
